<p align='justify'>
I am an [E.W. Schmidt Center Postdoctoral Fellow](https://www.ericandwendyschmidtcenter.org/) at the [Broad Institute of MIT and Harvard](https://www.broadinstitute.org/). My research is at the intersection of **computer science**, **molecular biology**, and **statistics**. I develop algorithms and machine learning models to integrate multi-modal genomic data, with a goal of understanding cellular dynamics in health and disease. I received my Ph.D. in computer science and computational biology from **Brown University**, under the advisement of [Ritambhara Singh, Ph.D.](http://rsinghlab.org/) (primary advisor) and [Sorin Istrail, Ph.D.](https://www.brown.edu/Research/Istrail_Lab/sorin.php) My [doctoral dissertation](https://repository.library.brown.edu/studio/item/bdr:er5vz2ca/) was on optimal transport algorithms for integrated analysis of single-cell multi-omics data. </p> 
<p align='justify'>
Since January 2024, I co-chair the [Models, Inference, and Algorithms (MIA)](https://www.broadinstitute.org/talks/fall-2023/mia) seminar series at the Broad, where we host hybrid (in-person & online) talks by computational scientists in bio(medicine). You can check out our [upcoming schedule](https://www.broadinstitute.org/talks/fall-2023/mia) and [past talks](https://www.youtube.com/watch?v=zE_G9rjIhEQ&list=PLlMMtlgw6qNjROoMNTBQjAcdx53kV50cS).  </p>
<!-- I am honored to be recognized as part of the [Rising Stars in EECS 2022 cohort](https://risingstars.utexas.edu/#:~:text=Rising%20Stars%20is%20an%20intensive,artificial%20intelligence%20and%20decision%2Dmaking.) by The University of Texas at Austin.<br>
 --><!-- 
Before joining Brown, I received my bachelor’s degree in **bioengineering** from **Olin College of Engineering**. There, I worked with [Jean J. Huang, Ph.D.](http://www.olin.edu/faculty/profile/jean-j-huang/) and [John Geddes, Ph.D.](http://www.olin.edu/faculty/profile/john-b-geddes) on bioinformatic analyses and dynamical modeling of microbial communities under various perturbations. Later, I worked in the [Quantitative Biology Lab](http://gwli.scripts.mit.edu/group/?p=737), led by [Gene-Wei Li, Ph.D.](https://biology.mit.edu/profile/gene-wei-li/) at the **Massachusetts Institute of Technology (MIT)**, as a research support associate and lab manager, exploring gene regulatory network rewiring in bacteria. In the summers of 2020 and 2022, I interned with two [Health Futures groups](https://www.microsoft.com/en-us/research/lab/microsoft-health-futures/) at **Microsoft Research.** <br> -->
<p align='justify'>
Apart from research, I have had the opportunity to work on various fun projects that can be found [here](/projects/). [Outside of work](/fun/), I enjoy boardgames with friends, swimming, playing with my cat, and practicing the ukulele and violin. Feel free to [e-mail me](mailto:pinardemetci@gmail.com) if you'd like to talk. <br>
</p>
<style>
.grid-container {
  display: grid;
  grid-template-columns: 450px 450px;
}

.grid-item {
  border:0;
}

div.interest-container>ul {
  display: grid; 
  grid-auto-columns: 1fr; 
  grid-auto-rows: 1fr; 
  grid-template-columns: 1fr 1fr; 
  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr; 
  column-gap: 0px; 
  grid-template-areas: 
    "col1-item1 col2-item1"
    "col1-item2 col2-item2"
    "col1-item3 col2-item3"
    "col1-item4 col2-item4" 
    "col1-item5 col2-item5"
    "col1-item6 col2-item6"
    "col1-item7 col2-item7"
    "col1-item8 col2-item8"
    "col1-item9 col2-item9";
}

/* Assign a class to each li */
.col1-item1 { grid-area: col1-item1; }
.col1-item2 { grid-area: col1-item2; }
.col1-item3 { grid-area: col1-item3; }
.col1-item4 { grid-area: col1-item4; }
.col1-item5 { grid-area: col1-item5; }
.col1-item6 { grid-area: col1-item6; }
.col1-item7 { grid-area: col1-item7; }
.col1-item8 { grid-area: col1-item8; }

.col2-item1 { grid-area: col2-item1; }
.col2-item2 { grid-area: col2-item2; }
.col2-item3 { grid-area: col2-item3; }
.col2-item4 { grid-area: col2-item4; }
.col2-item5 { grid-area: col2-item5; }
.col2-item6 { grid-area: col2-item6; }
.col2-item7 { grid-area: col2-item7; }
</style>

#### <i class="fa fa-chevron-right"></i> Research Interests

<div class="grid-container">
  <div class="grid-item"> <h5> Methodology </h5></div>
  <div class="grid-item"> <h5> Application areas </h5></div>
</div>

<div class="interest-container">
<ul>
 
  <li class="col1-item1">Representation learning</li>
  <li class="col1-item2">Optimal transport</li>
  <li class="col1-item3">Manifold learning</li>
  <li class="col1-item4">Bayesian statistics and inference</li>
  <li class="col1-item5">Variable selection</li>
  <li class="col1-item6">Deep learning</li>
  <li class="col1-item7">Causality</li>
  <li class="col1-item8">Graph algorithms</li>

  <li class="col2-item1">Regulatory genomics</li>
  <li class="col2-item2">Functional genomics</li>
  <li class="col2-item3">Single-cell sequencing & imaging</li>
  <li class="col2-item4">Multi-omics</li>
  <li class="col2-item5">3D genome</li>
  <li class="col2-item6">Precision medicine</li>
  <li class="col2-item7">Structural biology</li>
</ul>
</div>

#### <i class="fa fa-chevron-right"></i> Education

<table class="table table-hover">
  <tr>
    <td class="col-md-3"> 2018 - 2023 </td>
    <td>
        <strong>Ph.D. in Computer Science and Computational Biology </strong>
          (3.90/4.00)
        <br>
        <strong>M.Sc. in Computer Science</strong>
          (4.00/4.00)
        <br>
      Brown University (Providence, RI)
    </td>
  </tr>
  <tr>
    <td class="col-md-3"> 2013 - 2017</td>
    <td>
        <strong>B.Sc. in Engineering</strong> (with concentration in <strong>Bioengineering</strong>)
          (3.67/4.00)
        <br>
      Olin College of Engineering (Needham, MA)
    </td>
  </tr>
  <tr>
    <td class="col-md-3"> 2008 - 2013</td>
    <td>
      TEVITOL High School with IB Diploma (Gebze, Turkey)
    </td>
  </tr>
</table>

#### <i class="fa fa-chevron-right"></i> Professional Experience
<table class="table table-hover"><tr>
  <td class='col-md-3'>July 2023 - Present</td>
  <td><strong>Broad Institute of MIT and Harvard</strong>, EWSC Postdoctoral Fellow (Cambridge, MA)</td>
</tr>
<tr>
</tr>
<tr>
  <td class='col-md-3'>June 2022 - Aug 2022</td>
  <td><strong>Microsoft Research</strong>, Research Intern (Redmond, WA)</td>
</tr>
<tr>
</tr>
<tr>
  <td class='col-md-3'>June 2020 - Sep 2020</td>
  <td><strong>Microsoft Research</strong>, Research Intern (Redmond, WA)</td>
</tr>
<tr>
</tr>
<tr>
  <td class='col-md-3'>June 2017 - August 2018</td>
  <td><strong>Massachusetts Institute of Technology</strong>, Research Support Associate (Cambridge, MA)</td>
</tr>
<tr>
  <td class='col-md-3'>Jan 2016 - Oct 2016</td>
  <td><strong>Design That Matters</strong>, Student Engineer (Salem, MA)</td>
</tr>
<tr>
  <td class='col-md-3'>Jan 2015 - Dec 2015</td>
  <td><strong>Daktari Diagnostics</strong>, Student Engineer (Cambridge, MA)</td>
</tr>
</table>

#### <i class="fa fa-chevron-right"></i> Research Publications <a href="https://scholar.google.com/citations?user=0Tzd6eAAAAAJ&hl=en" class="btn btn-primary" style="padding: 0.3em;"><i class="ai ai-google-scholar"></i> Google Scholar </a>
<h4>2024</h4>
<table class="table table-hover">
<tr>
<td class="col-md-3"><a href='https://link.springer.com/chapter/10.1007/978-3-031-04749-7_1' target='_blank'><img src="images/publications/agw.png"/></a> </td>
<td>
    <strong> Breaking isometric ties and introducing priors in Gromov-Wasserstein distances</strong><br>
    <strong>P. Demetci</strong>, Q.H. Tran, I. Redko, R. Singh<br>
    Proceedings of the 27th International Conference on Artificial Intelligence and Statistics <br>
    <em>(AISTATS 2024)</em><br>
    Proceedings of Machine Learning Research (PMLR) -- to appear<br>
    [11] 
[<a href='javascript:;'
    onclick='$("#abs_demetciagw").toggle()'>abstract</a>] [<a href='https://arxiv.org/abs/2307.10093' target='_blank'>paper</a>] [<a href='https://github.com/pinardemetci/AGW-AISTATS24' target='_blank'>code</a>] <br>
    
<div id="abs_demetciagw" style="text-align: justify; display: none" markdown="1">
Gromov-Wasserstein distance has many applications in machine learning due to its ability to compare measures across metric spaces and its invariance to isometric transformations. However, in certain applications, this invariant property can be too flexible, thus undesirable. Moreover, the Gromov-Wasserstein distance solely considers pairwise sample similarities in input datasets, disregarding the raw feature representations. We propose a new optimal transport formulation, called Augmented Gromov-Wasserstein (AGW), that allows for some control over the level of rigidity to transformations. It also incorporates feature alignments, enabling us to better leverage prior knowledge on the input data for improved performance. We present theoretical insights into the proposed method. We then demonstrate its usefulness for single-cell multi-omic alignment tasks and heterogeneous domain adaptation in machine learning.
</div>
</td>
</tr>
</table>
<h4>2023</h4>
<table class="table table-hover">
<tr>
<td class="col-md-3"><a href='https://arxiv.org/abs/2205.14923' target='_blank'><img src="images/publications/ucoot.png"/></a> </td>
<td>
    <strong>Unbalanced CO-Optimal Transport</strong><br>
    Q.H. Tran, H. Janati, N. Courty, R. Flamary, I. Redko, <strong>P. Demetci</strong>,  R Singh<br>
    Proceedings of the 37th AAAI Conference on Artificial Intelligence<br>
    <em>(AAAI 2023)</em><br>
    [10] 
[<a href='javascript:;'
    onclick='$("#abs_ucoot").toggle()'>abstract</a>] [<a href='https://ojs.aaai.org/index.php/AAAI/article/view/26193/25965' target='_blank'>paper</a>] [<a href='https://github.com/6Ulm/UCOOT' target='_blank'>code</a>] <br>
    
<div id="abs_ucoot" style="text-align: justify; display: none" markdown="1">
Optimal transport (OT) compares probability distributions by computing a meaningful alignment between their samples. CO-optimal transport (COOT) takes this comparison further by inferring an alignment between features as well. While this approach leads to better alignments and generalizes both OT and Gromov-Wasserstein distances, we provide a theoretical result showing that it is sensitive to outliers that are omnipresent in real-world data. This prompts us to propose unbalanced COOT for which we provably show its robustness to noise in the compared datasets. To the best of our knowledge, this is the first such result for OT methods in incomparable spaces. With this result in hand, we provide empirical evidence of this robustness for the challenging tasks of heterogeneous domain adaptation with and without varying proportions of classes and simultaneous alignment of samples and features across single-cell measurements.
</div>
</td>
</tr>

<tr>
<td class="col-md-3"><a href='https://arxiv.org/abs/2205.14923' target='_blank'><img src="images/publications/neuro2.png"/></a> </td>
<td>
    <strong>Mammalian olfactory cortex neurons retain molecular signatures of ancestral cell types</strong><br>
    S. Zeppilli, A. Ortega Gurrola, <strong>P. Demetci</strong>, DH. Brann, R. Attey, N. Zilkha, T. Kimchi, SR. Datta, R. Singh, MA. Tosches, A. Crombach, A. Fleischmann<br>
    <em>bioRxiv (Under review at Nature Neuroscience)</em> <br>
    [9] 
[<a href='javascript:;'
    onclick='$("#abs_zeppilli").toggle()'>abstract</a>] [<a href='https://www.biorxiv.org/content/10.1101/2023.08.13.553130v1.abstract' target='_blank'>paper</a>] [<a href='https://gitlab.com/fleischmann-lab/papers/zeppilli-et-al-2023' target='_blank'>code</a>] <br>
    
<div id="abs_zeppilli" style="text-align: justify; display: none" markdown="1">
The cerebral cortex diversified extensively during vertebrate evolution. Intriguingly, the three-layered mammalian olfactory cortex resembles the cortical cytoarchitecture of non-mammals yet evolved alongside the six-layered neocortex, enabling unique comparisons for investigating cortical neuron diversification. We performed single-nucleus multiome sequencing across mouse three- to six-layered cortices and compared neuron types across mice, reptiles and salamander. We identified neurons that are olfactory cortex-specific or conserved across mouse cortical areas. However, transcriptomically similar neurons exhibited area-specific epigenetic states. Additionally, the olfactory cortex showed transcriptomic divergence between lab and wild-derived mice, suggesting enhanced circuit plasticity through adult immature neurons. Finally, olfactory cortex neurons displayed marked transcriptomic similarities to reptile and salamander neurons. Together, these data indicate that the mammalian olfactory cortex retains molecular signatures representative of ancestral cortical traits.
</div>
</td>
</tr>
</table>

<h4>2022</h4>
<table class="table table-hover">
<tr>
<td class="col-md-3"><a href='https://link.springer.com/chapter/10.1007/978-3-031-04749-7_1' target='_blank'><img src="images/publications/scotv2.png"/></a> </td>
<td>
    <strong> Unsupervised Integration of Single-Cell Multi-omics Datasets with Disproportionate Cell-Type Representation</strong><br>
    <strong>P. Demetci</strong>, R. Santorella, B. Sandstede, R. Singh<br>
    Proceedings of the 26th Annual International Conference on Research in Computational Molecular Biology<br>
    <em>(RECOMB 2022)</em><br>
    Springer Nature Lecture Notes in Bioinformatics (2022) pp 3-19<br>
    [8] 
[<a href='javascript:;'
    onclick='$("#abs_demetciscot2").toggle()'>abstract</a>] [<a href='https://link.springer.com/chapter/10.1007/978-3-031-04749-7_1' target='_blank'>paper</a>] [<a href=' http://rsinghlab.github.io/SCOT' target='_blank'>code</a>] <br>
    
<div id="abs_demetciscot2" style="text-align: justify; display: none" markdown="1">
Integrated analysis of multi-omics data allows the study of how different molecular views in the genome interact to regulate cellular processes; however, with a few exceptions, applying multiple sequencing assays on the same single cell is not possible. While recent unsupervised algorithms align single-cell multi-omic datasets, these methods have been primarily benchmarked on co-assay experiments rather than the more common single-cell experiments taken from separately sampled cell populations. Therefore, most existing methods perform subpar alignments on such datasets. Here, we improve our previous work Single Cell alignment using Optimal Transport (SCOT) by using unbalanced optimal transport to handle disproportionate cell-type representation and differing sample sizes across single-cell measurements. We show that our proposed method, SCOTv2, consistently yields quality alignments on five real-world single-cell datasets with varying cell-type proportions and is computationally tractable. Additionally, we extend SCOTv2 to integrate multiple (M ≥ 2) single-cell measurements and present a self-tuning heuristic process to select hyperparameters in the absence of any orthogonal correspondence information.
</div>
</td>
</tr>
</table>

<h4>2021</h4>
<table class="table table-hover">
 <tr>
<td class="col-md-3"><a href='' target='_blank'><img src="images/publications/SCOT_JCB.png"/></a> </td>
<td>
    <strong>SCOT: Single-cell multi-omics integration with optimal transport</strong><br>
    <strong>P. Demetci*</strong>, R. Santorella*, B. Sandstede, W. Stafford Noble and Ritambhara Singh# <br>
     *Equal Contribution, #Corresponding Author<br>
    <em>Journal of Computational Biology, 2021</em><br>
    [7] 
[<a href='javascript:;'
    onclick='$("#abs_demetcisantorella").toggle()'>abstract</a>] [<a href='https://www.biorxiv.org/content/10.1101/2020.04.28.066787v1' target='_blank'>paper</a>] [<a href='https://github.com/rsinghlab/SCOT' target='_blank'>code</a>] [<a href='https://rsinghlab.github.io/SCOT/' target='_blank'>tutorial</a>] <br>
    
<div id="abs_demetcisantorella" style="text-align: justify; display: none" markdown="1">
Recent advances in sequencing technologies have allowed us to capture various aspects of the genome at single-cell resolution. However, with the exception of a few of co-assaying technologies, it is not possible to simultaneously apply different sequencing assays on the same single cell. In this scenario, computational integration of multi-omic measurements is crucial to enable joint analyses. This integration task is particularly challenging due to the lack of sample-wise or feature-wise correspondences. We present Single-Cell alignment with Optimal Transport (SCOT), an unsupervised algorithm that uses Gromov-Wasserstein optimal transport to align single-cell multi-omics datasets. SCOT performs on par with the current state-of-the-art unsupervised alignment methods, is faster, and requires tuning of fewer hyperparameters. More importantly, SCOT uses a self-tuning heuristic to guide hyperparameter selection based on Gromov-Wasserstein distance. Thus, in the fully unsupervised setting, SCOT aligns single-cell datasets better than the existing methods without requiring any orthogonal correspondence information. </a>
</div>
</td>
</tr>

<tr>
<td class="col-md-3"><a href='https://www.biorxiv.org/content/10.1101/2020.04.28.066787v1' target='_blank'><img src="images/publications/SCOT.png"/></a> </td>
<td>
    <strong>Gromov-Wasserstein Optimal Transport to Align Single-Cell Multi-Omics Data</strong><br>
    <strong>P. Demetci*</strong>, R. Santorella*, B. Sandstede, W. Stafford Noble and Ritambhara Singh# <br>
     *Equal Contribution, #Corresponding Author<br>
    International Conference on Research in Computational Molecular Biology<br>
    <em>(RECOMB 2021)</em><br>
    [6] 
[<a href='javascript:;'
    onclick='$("#abs_demetcisantorella").toggle()'>abstract</a>] [<a href='https://www.biorxiv.org/content/10.1101/2020.04.28.066787v1' target='_blank'>paper</a>] [<a href='https://github.com/rsinghlab/SCOT' target='_blank'>code</a>] [<a href='https://rsinghlab.github.io/SCOT/' target='_blank'>tutorial</a>] <br>
    
<div id="abs_demetcisantorella" style="text-align: justify; display: none" markdown="1">
Data integration of single-cell measurements is critical for our understanding of cell development and disease, but the lack of correspondence between different types of single-cell measurements makes such efforts challenging. Several unsupervised algorithms are capable of aligning heterogeneous types of single-cell measurements in a shared space, enabling the creation of mappings between single cells in different data modalities.
We present Single-Cell alignment using Optimal Transport (SCOT), an unsupervised learning algorithm that uses Gromov Wasserstein-based optimal transport to align single-cell multi-omics datasets. SCOT calculates a probabilistic coupling matrix that matches cells across two datasets. The optimization uses k-nearest neighbor graphs, thus preserving the local geometry of the data. We use the resulting coupling matrix to project one single-cell dataset onto another via a barycentric projection.  We compare the alignment performance of SCOT with state-of-the-art algorithms on three simulated and two real datasets. Our results demonstrate that SCOT yields results that are comparable in quality to those of competing methods, but SCOT is significantly faster and requires tuning fewer hyperparameters. The code is available at <a href='https://github.com/rsinghlab/SCOT' target='_blank'>https://github.com/rsinghlab/SCOT</a>
</div>
</td>
</tr>

<tr>
<td class="col-md-3"><a href='https://www.biorxiv.org/content/10.1101/2020.07.02.184465v3' target='_blank'><img src="images/publications/bann.png"/></a> </td>
<td>
    <strong>Multi-scale Inference of Genetic Trait Architecture using Biologically Annotated Neural Networks</strong><br>
    <strong>P. Demetci</strong>,W. Cheng,Gregory Darnell, Xiang Zhou, Sohini Ramachandran, Lorin Crawford# <br>
    <em>PLOS Genetics, 2021</em><br>
    [5] 
[<a href='javascript:;'
    onclick='$("#abs_bann").toggle()'>abstract</a>] [<a href='https://www.biorxiv.org/content/10.1101/2020.07.02.184465v3' target='_blank'>paper</a>] <br>
    
<div id="abs_bann" style="text-align: justify; display: none" markdown="1">
In this article, we present Biologically Annotated Neural Networks (BANNs), a nonlinear probabilistic framework for association mapping in genome-wide association (GWA) studies. BANNs are feedforward models with partially connected architectures that are based on biological annotations. This setup yields a fully interpretable neural network where the input layer encodes SNP-level effects, and the hidden layer models the aggregated effects among SNP-sets. We treat the weights and connections of the network as random variables with prior distributions that reflect how genetic effects manifest at different genomic scales. The BANNs software uses variational inference to provide posterior summaries which allow researchers to simultaneously perform (i) mapping with SNPs and (ii) enrichment analyses with SNP-sets on complex traits. Through simulations, we show that our method improves upon state-of-the-art association mapping and enrichment approaches across a wide range of genetic architectures. We then further illustrate the benefits of BANNs by analyzing real GWA data assayed in approximately 2,000 heterogenous stock of mice from the Wellcome Trust Centre for Human Genetics and approximately 7,000 individuals from the Framingham Heart Study. Lastly, using a random subset of individuals of European ancestry from the UK Biobank, we show that BANNs is able to replicate known associations in high and low-density lipoprotein cholesterol content.</a>
</div>

</td>
</tr>
</table>

<h4>2020</h4>
<table class="table table-hover">
<tr>
<td class="col-md-3"><a href='https://www.biorxiv.org/content/10.1101/2020.06.13.149195v1' target='_blank'><img src="images/publications/alignment.png"/></a> </td>
<td>
    <strong> Unsupervised Manifold Alignment for Single-Cell Multi-Omics Data</strong><br>
    R. Singh#, <strong>P. Demetci</strong>, G. Bonora, V. Ramani, C. Lee, H. Fang, Z. Duan, X. Deng, J. Shendure, C. Disteche and W. Stafford Noble#<br>
     #Corresponding Authors<br>
    Proceedings of the 11th ACM International Conference on Bioinformatics, Computational Biology, and Health Informatics<br>
    <em>(ACM BCB 2020)</em><br>
    
    [4] 
[<a href='javascript:;'
    onclick='$("#abs_singhdemetci").toggle()'>abstract</a>] [<a href='https://dl.acm.org/doi/10.1145/3388440.3412410' target='_blank'>paper</a>] <a href='https://bitbucket.org/noblelab/2020_mmdma_pytorch/src/master/' target='_blank'>[code]</a> <br>
    
<div id="abs_singhdemetci" style="text-align: justify; display: none" markdown="1">
Integrating single-cell measurements that capture different properties of the genome is vital to extending our understanding of genome biology.This task is challenging due to the lack of a shared axis across datasets obtained from different types of single-cell experiments. For most such datasets, we lack corresponding information among the cells (samples) and the measurements (features). In this scenario, unsupervised algorithms that are capable of aligning single-cell experiments are critical to learning an in silico co-assay that can help draw correspondences among the cells.Maximum mean discrepancy-based manifold alignment (MMD-MA) is such an unsupervised algorithm. Without requiring correspondence information, it can align single-cell datasets from different modalities in a common shared latent space, showing promising results on simulations and a small-scale single-cell experiment with 61 cells.However, it is essential to explore the applicability of this method to larger single-cell experiments with thousands of cells so that it can be of practical interest to the community.In this paper, we apply MMD-MA to two recent datasets that measure transcriptome and chromatin accessibility in ~2000 single cells. To scale the runtime of MMD-MA to a more substantial number of cells, we extend the original implementation to run on GPUs. We also introduce a method to automatically select one of the user-defined parameters, thus reducing the hyperparameter search space. We demonstrate that the proposed extensions allow MMD-MA to accurately align state-of-the-art single-cell experiments.
</div>

</td>
</tr>


<tr>
<td class="col-md-3"><a href='https://www.iscb.org/cms_addon/conferences/ismb2020/proceedings.php' target='_blank'><img src="images/publications/haplex_new.png"/></a> </td>
<td>
    <strong>Combinatorial and statistical prediction of gene expression from haplotype sequence</strong><br>
    B. Alpay*, <strong>P. Demetci*</strong> Sorin Istrail, and Derek Aguiar# <br>
     *Equal Contribution, #Corresponding Author<br>
     Bioinformatics (Oxford Press) vol.36, Supplement_1, p:i194-i202. 2020 <br>
    Proceedings of the 27th International Conference on Intelligent Systems for Molecular Biology<br>
    <em>(ISMB 2020)</em><br>
    
    [3] 
[<a href='javascript:;'
    onclick='$("#abs_alpaydemetci").toggle()'>abstract</a>] [<a href='https://www.iscb.org/cms_addon/conferences/ismb2020/proceedings.php' target='_blank'>paper</a>] <a href='https://github.com/rapturous/HAPLEX' target='_blank'>[code]</a> <br>
    
<div id="abs_alpaydemetci" style="text-align: justify; display: none" markdown="1">
Genome-wide association studies (GWAS) have discovered thousands of significant genetic effects on disease phenotypes.By considering gene expression as the intermediary between genotype and disease phenotype, eQTL studies have interpreted many of these variants by their regulatory effects on gene expression. However, there remains a considerable gap between genotype-to-gene expression association and genotype-to-gene expression prediction. Accurate prediction of gene expression enables gene-based association studies to be performed post-hoc for existing GWAS, reduces multiple testing burden, and can prioritize genes for subsequent experimental investigation.In this work, we develop gene expression prediction methods that relax the independence and additivity assumptions between genetic markers. First, we consider gene expression prediction from a regression perspective and develop the HAPLEXR algorithm which combines haplotype clusterings with allelic dosages. Second, we introduce the new gene expression classification problem, which focuses on identifying expression groups rather than continuous measurements; we formalize the selection of an appropriate number of expression groups using the principle of maximum entropy. Third, we develop the HAPLEXD algorithm that models haplotype sharing with a modified suffix tree data structure and computes expression groups by spectral clustering. In both models, we penalize model complexity by prioritizing genetic clusters that indicate significant effects on expression. We compare HAPLEXR and HAPLEXD with three state-of-the-art expression prediction methods and two novel logistic regression approaches across five GTEx v8 tissues. HAPLEXD exhibits significantly higher classification accuracy overall; HAPLEXR shows higher prediction accuracy on approximately half of the genes tested and the largest number of best predicted genes ($r^2>0.1$) among all methods.
We show that variant and haplotype features selected by HAPLEXR are smaller in size than competing methods (and thus more interpretable) and are significantly enriched in functional annotations related to gene regulation. These results demonstrate the importance of explicitly modelling non-dosage dependent and intragenic epistatic effects when predicting expression.
</div>

</td>
</tr>
</table>

<h4>2019</h4>
<table class="table table-hover">
<tr>
<td class="col-md-3"><a href='https://jb.asm.org/content/early/2019/07/03/JB.00259-19/figures-only?versioned=true' target='_blank'><img src="images/publications/ecoli.png"/></a> </td>
<td>
    <strong>Rapid accumulation of motility-activating mutations in resting liquid culture of <em>Escherichia coli</em></strong><br>
    D. Parker*, <strong>P.Demetci*</strong>, and G.W. Li# <br>
     *Equal Contribution,  #Corresponding Author<br>
    <em>Journal of Bacteriology, 2019</em><br>
    
    [2] 
[<a href='javascript:;'
    onclick='$("#abs_parkerdemetci").toggle()'>abstract</a>] [<a href='https://jb.asm.org/content/early/2019/07/03/JB.00259-19/figures-only?versioned=true' target='_blank'>paper</a>] <br>
    
<div id="abs_parkerdemetci" style="text-align: justify; display: none" markdown="1">
Expression of motility genes is a potentially beneficial but costly process in bacteria. Interestingly, many isolate strains of _Escherichia coli_ possess motility genes but have lost the ability to activate them in conditions in which motility is advantageous, raising the question of how they respond to these situations. Through transcriptome profiling of strains in the _E. coli_ single-gene knockout Keio collection, we noticed drastic upregulation of motility genes in many of the deletion strains as compared to its weakly motile parent strain (BW25113). We show that this switch to a motile phenotype is not a direct consequence of the genes deletec, but is instead due to a variety of secondary mutations that increase the expression of the major motility regulator, FlhDC. Importantly, we find that this switch can be reproduced by growing poorly motile _E. coli_ strains in non-shaking liquid medium overnight but not in shaking liquid medium. Individual isolates after the non-shaking overnight incubations acquired distinct mutations upstream of the _flhDC_ operon, including different insertion sequence (IS) elements and, to a lesser extent, point mutations. The rapid sweep in the non-shaking population shows that poorly motile strains can quickly adapt to a motile lifestyle by genetic rewiring. 
</div>
</td>
</tr>
</table>

<h4>2016</h4>
<table class="table table-hover">
<tr>
<td class="col-md-3"><a href='https://ieeexplore.ieee.org/abstract/document/7757463' target='_blank'><img src="images/publications/blank.png"/></a> </td>
<td>
    <strong>Internalization and externalization in the classroom: How do they emerge and why is it important?</strong><br>
     <strong>P.Demetci</strong>, C. Nichols, Y. V. Zastavker, J. D. Stolk, A. Dillon, M. D. Gross.<br>
    IEEE Frontiers in Education<br>
    <em>(FIE 2016)</em><br>
    
    [1] 
[<a href='javascript:;'
    onclick='$("#abs_demetcinichols").toggle()'>abstract</a>] [<a href='https://ieeexplore.ieee.org/abstract/document/7757463'  target='_blank'>paper</a>] <br>  
<div id="abs_demetcinichols" style="text-align: justify; display: none" markdown="1">
“I felt so dumb, and it's not fair that I cannot grasp this information to save my life, and other people can with no problem.” Why do some students feel empowered in the classroom, and feel they have control over their own learning, while others do not? Our qualitative investigation is a part of a larger mixed-methods study about students' situational motivations in introductory STEM courses. We used grounded theory to analyze students' responses to surveys about emotion, course relevance, and motivation. We investigated two emergent phenomena we called “internalization” and “externalization.” Our definitions of these are based on a student's perception of who or what influences the outcomes of their activities: the students themselves, or external factors such as the instructor, peers, or the educational system as a whole. Our findings indicate that (1) internalization correlates with cognitive autonomy, students' perception of course content having high personal relevance, and group projects in project-based learning environments; and (2) externalization correlates with lecture-based environments and students' perceptions of lack of personal relevance in the course content. Our analyses suggest that project-based learning environments may serve to empower students, but only when course content is found to be relevant.
</div>
</td>
</tr>
</table>

#### <i class="fa fa-chevron-right"></i> Awards and Honors
<table class="table table-hover">
  <tr>
  <td class='col-md-2'>2023</td>
  <td>
  <a href="https://www.ericandwendyschmidtcenter.org/">EWSC Postdoctoral Fellowship</a> <em>(up to three years)</em>
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2023</td>
  <td>
  <a href="https://ds.dfci.harvard.edu/postdocs/">Harvard/DFCI Data Science Postdoctoral Fellowship</a> <em>(turned down)</em>
    <!--  -->
  </td>
</tr>
  <tr>
  <td class='col-md-2'>2022</td>
  <td>
   Selected to the <a href="https://risingstars.utexas.edu/">Rising Stars in EECS cohort</a> <em>(by UT Austin)</em>
    <!--  -->
  </td>
</tr>
 <tr>
  <td class='col-md-2'>2022</td>
  <td>
   RECOMB Travel Fellowship
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2020</td>
  <td>
   ICML WCB Fellowship
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2020</td>
  <td>
  Microsoft Research Ph.D. Fellowship Nominee <em>(by Brown CCMB)</em>
    <!--  -->
  </td>
</tr>

<!-- <tr>
  <td class='col-md-2'>2020</td>
  <td>
   ICML WCB Best Poster
  </td>
</tr> -->
<tr>
  <td class='col-md-2'>2016</td>
  <td>
    <a href="https://www.comap.com/undergraduate/contests/mcm/contests/2016/results/2016-ICM_Problem-F-Results.pdf">Meritorius Winner: 2016 MCM/ICM Interdisciplinary Contest in Mathematical Modeling</a>
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2015-2017</td>
  <td>
    Olin Alumni Scholarship <em>(towards expenses beyond tuition)</em>
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2013-2017</td>
  <td>
    <a href="http://impact.olin.edu/2017/scholarships#scholarships">Sunlin Chou International Scholarship</a> <em>(50% tuition)</em>
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2013-2017</td>
  <td>
    Olin Merit Scholarship <em>(50% tuition)</em>
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2013</td>
  <td>
    <a href="http://info.ifpan.edu.pl/firststep/results/21">Honorable Mention (Instrumentation):  21st Intl. Competition of First Step to Nobel Prize in Physics</a><br> <em>(by the Polish Academy of Sciences)</em>
    <!--  -->
  </td>
</tr>
<tr>
  <td class='col-md-2'>2013</td>
  <td>
    <a href="http://info.ifpan.edu.pl/firststep/results/21">First Place in Physics: 22nd MEF International Research Projects Contest</a> <em>(Turkiye)</em>
    <!--  -->
  </td>
</tr>
</table>
